src/optimization/optimizerInterface.cpp

```diff
@@ -49,7 +49,6 @@ double optimizerInterface::get (const std::string & /*param*/) const
 
 int optimizerInterface::check_flag (void *flagvalue, const std::string &funcname, int opt, bool printError)
```

Comment:
this function is a replicate of what was in solverInterfaces,  it probably isn't needed here since we are updating the optimizer interface.